content/articles/image-compression-using-PCA/index.md

```diff
@@ -0,0 +1,336 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /image-compression-using-PCA/
+title: Image Compression Using PCA
+description: In this article we discuss the dimensionality reduction algorithm, Principal Component Analysis and use the same to compress images.
+author: deewakar-chakraborty
+date: 2021-06-10T00:00:00-18:00
+topics: [Machine Learning]
+excerpt_separator: <!--more-->
+images:
+
+  -url: /engineering-education/image-compression-using-PCA/hero.png
+   alt: Image compression using PCA example image
+---
+
+Principal Component Analysis, PCA, is a dimensionality reduction method used to reduce the dimensionality of a dataset by transforming the data to a new basis where the dimensions are non-redundant(low covariance) and have high variance. This tutorial aims to make the reader understand the concept of PCA(Principal Component Analysis) mathematically by providing them with one of the use cases of PCA, image compression.
+
+### Table of Contents
+
+- Prerequisites.
+- Introduction
+  -  What is Dimensionality Reduction?
+  - Motivation for Dimensionality Reduction.
+  - Why not care about u<sub>2</sub>.
+  - Correlation.
+  - Desideratum.
+
+- Principal Component Analysis(PCA)
+  -  Orthogonal Transformation.
+  - Covariance Matrix.
+  - What do we want from the covariance matrix of transformed data?
+  - Few points to ponder.
+  - Principal Components.
+  - Dimensionality Reduction.
+
+- Use case: Image Compression
+  - How is an image compressed?
+  - Reconstructing images using less information.
+
+- Conclusion.
+
+### Prerequisites
+
+The reader needs to have basic knowledge of algebra(matrix operations and their properties) and statistics.
+
+### Introduction
+
+#### What is Dimensionality Reduction?
+
+<b>Dimensionality</b>: The number of input variables or features for a dataset is referred to as its dimensionality.
+
+> **Dimensionality reduction** is the transformation of data from a high-dimensional space into a low-dimensional space so that the low-dimensional representation retains some meaningful properties of the original data, ideally close to its intrinsic dimension(number of variables needed in a minimal representation of the data).
+
+Dimensionality reduction refers to techniques that reduce the number of input variables in a dataset. More input features often make a predictive modeling task more challenging to model, generally referred to as the <b>curse of dimensionality</b>.
+
+To explain the concept of dimensionality reduction, I will take an example, consider the following data where each point(vector) is represented using a linear combination of the **x** and **y** axes:
+
+<center>
+
+
+![first-graph](plot-1.png)
```

Comment:
All images go with the following URL:

\[alt text\]\(/engineering-education/image-compression-using-PCA/image.type\)